packages/contracts-bedrock/src/cannon/MIPS2.sol

```diff
@@ -162,8 +162,11 @@ contract MIPS2 is ISemver {
                 return outputState();
             }
 
-            if (state.leftThreadStack == EMPTY_THREAD_ROOT && state.rightThreadStack == EMPTY_THREAD_ROOT) {
-                revert("MIPS2: illegal vm state");
+            if (
```

Comment:
A couple things related to this change:
- Contract changes require a version bump.  The MIPS2 version logic is [here](https://github.com/ethereum-optimism/optimism/blob/18732dbafa6836d7ec6d5adaaf45108c29454fc8/packages/contracts-bedrock/src/cannon/MIPS2.sol#L59-L61).  You can just increment the patch number by 1.  Details on versioning are [here](https://github.com/ethereum-optimism/optimism/blob/develop/packages/contracts-bedrock/VERSIONING.md) if you're curious.
- You need to rerun some `just` tasks to get the build passing since there are contract changes.  Details on that [here](https://github.com/ethereum-optimism/optimism/blob/18732dbafa6836d7ec6d5adaaf45108c29454fc8/packages/contracts-bedrock/CONTRIBUTING.md#your-first-code-contribution).  I think you just need to run `just pre-pr` after installing `just`.

Reply:
appreciate for the tips!